.env.example

```diff
@@ -7,3 +7,7 @@ GOOGLE_CLIENT_ID=""
 GOOGLE_CLIENT_SECRET=""
 
 DATABASE_URL="postgres://postgres:postgres@localhost:5432/postgres"
+
+SMTP_HOST="smtp.gmail.com"
```

Comment:
jeśli chcemy używać gmaila do wysyłania maili to z tego co kojarzę do logowania przy nim powinno się używać oautha, ale jeśli nie będzie problemów to możemy zostawić przez hasło

Reply:
Będę miał to na uwadze, jeżeli nie będzie działać to do tego wrócimy